salesforce_bulkipy/salesforce_bulkipy.py

```diff
@@ -1,15 +1,19 @@
 from __future__ import absolute_import
+from future.standard_library import install_aliases
+from future.utils import iteritems
+install_aliases()
+
 from tempfile import TemporaryFile
 from collections import namedtuple
 from httplib2 import Http
+from urllib.parse import urlparse
```

Comment:
Hello @bholagabbar,

We are from the same team and we are working on migrating this package to Python 3 as part of an internal migration initiative of our code base. @jeryini will close this PR and we will open another one from our company's official GitHub organization. Sorry for the hassle.

Cheers,
Ahmad